src/main/app/logging/customEventTracker.ts

```diff
@@ -1,10 +1,14 @@
 import * as appInsights from 'applicationinsights'
 
-export function trackCustomEvent (eventName: string, trackingProperties: {}): void {
-  if (appInsights.defaultClient) {
-    appInsights.defaultClient.trackEvent({
-      name: eventName,
-      properties: trackingProperties
-    })
+export function trackCustomEvent (eventName: string, trackingProperties: {}) {
+  try {
+    if (appInsights.defaultClient) {
+      appInsights.defaultClient.trackEvent({
+        name: eventName,
+        properties: trackingProperties
+      })
+    }
+  } catch (err) {
+    return
```

Comment:
Should we be doing something with this error than just returning?

Reply:
As above, have been removed